Scheduler/feature_scheduler/auxtel/jan_2025_fbs_spec_survey.py

```diff
@@ -89,36 +91,47 @@ def get_scheduler():
         ),
     ]
 
-    image_tiles = [
-        Tiles(
+    image_target_list = [
+        Target(
+            target_name="Photo0800-1",
             survey_name="BLOCK-306",
-            hour_angle_limit=image_ha_limit,
+            ra=Angle("08:10:00", unit=units.hourangle),
+            dec=Angle("-36:11:23", unit=units.deg),
+            hour_angle_limit=None,  # image_ha_limit,
             reward_value=reward_values["default"],
             filters=["r"],
             visit_gap=image_visit_gap,
             exptime=image_exptime,
             nexp=image_nexp,
-        ),
+        )
     ]
 
     spec_detailers = []
-    image_detailers = [
-        DitherDetailer(max_dither=(25.0 / (60.0 * 60.0)), per_night=True)
+    image_detailers_tile = [
+        # Tiny dither night-to-night. (does this work?)
```

Comment:
I think we can remove this comment. I've never compared the night-to-night pointings but it must be irrelevant for what we need since nobody else has brought it up.

Reply:
Do you actually just want me to remove the jan_2025_fbs_spec_survey.py updates altogether? They're not the way we ended up going, and would never have been used on-sky, so to me it makes sense to just let this die.

Reply:
Ah I see ... If I don't modify these files, to add the science_program to Target, then CI will start failing. 
However, you may consider a policy to start removing old configuration files (I see a few that are still around for auxtel), partly to make it easier to make updates in ts_fbs_utils without breaking unused configurations in ts_config_ocs. But I've updated them as needed here (I hope!).